tests/x11/yast2_snapper.pm

```diff
@@ -72,101 +31,26 @@ sub run() {
     # Start an xterm as root
     x11_start_program("xterm");
     assert_screen "xterm";
+    # Disable screen lock and blank screen for current Gnome session
+    script_run("gsettings set org.gnome.desktop.session idle-delay 0");
```

Comment:
Obviously it should not work for kde desktop :)
Looking for command for plasma 5...